### PR TITLE
Grid simplified

### PR DESCRIPTION
**Pull request by msdeshmukh009**
**Features added**
- Two-column (50:50) grid layout
- (30:70) grid layout
- (70:30) grid layout
- Three column grid layout

### CODE REVIEW

File: components/simplified-grid/simplified-grid.css [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  display: grid;
+  grid-template-columns: 30% 70%;
+}
+.grid-70-30 {
+  display: grid;
+  grid-template-columns: 70% 30%;
+}
+.grid-3-col {
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
```
Comment (sanathkumar26700) [inline components/simplified-grid/simplified-grid.css]: repeat(4,1fr) creates 4 columns so it has to be repeat(3, 1fr) for 3 columns.

**Review by sanathkumar26700 — APPROVED** [review]

File: components/simplified-grid/simplified-grid.css (hunk 1 above)
Comment (msdeshmukh009) [reply components/simplified-grid/simplified-grid.css]: Ohh I was trying something and forgot to remove it. Thanks for the review.

File: components/simplified-grid/simplified-grid.css [hunk 2]
```diff
@@ -0,0 +1,34 @@
+.grid-50-50 {
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
```
Comment (horsemaker) [inline components/simplified-grid/simplified-grid.css:3]: there's no `gap` here...

File: components/simplified-grid/simplified-grid.css [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+}
+.grid-30-70 {
+  display: grid;
+  grid-template-columns: 30% 70%;
```
Comment (horsemaker) [inline components/simplified-grid/simplified-grid.css:8]: same here, `gap` prop.

**Review by horsemaker — APPROVED** [review]

**Review by Chakravarthi0 — APPROVED** [review]
Looks good

**Review by MarkVed17 — APPROVED** [review]
Looks good to me!

**Pull request merged**